src/core/addonManager.js

```diff
@@ -123,15 +125,18 @@ export function install(
   return _mozAddonManager.createInstall({ url, hash }).then((installObj) => {
     const callback = (e) => eventCallback(installObj, e);
     for (const event of INSTALL_EVENT_LIST) {
-      log.info(`[install] Adding listener for ${event}`);
+      _log.info(`[install] Adding listener for ${event}`);
       installObj.addEventListener(event, callback);
     }
     return new Promise((resolve, reject) => {
       installObj.addEventListener('onInstallEnded', () => resolve());
       installObj.addEventListener('onInstallFailed', () => reject());
-      log.info('Events to handle the installation initialized.');
-      // See: https://github.com/mozilla/addons-frontend/issues/8633
-      installObj.install().catch(reject);
+      _log.info('Events to handle the installation initialized.');
+      installObj.install().catch((error) => {
+        // See: https://github.com/mozilla/addons-frontend/issues/8668
```

Comment:
I suggest also explaining how we rely on `onInstallFailed` or `onInstallEnded` for the promise resolution.